tests/conftest.py

```diff
@@ -32,13 +32,25 @@
 
 
 def pytest_collection_modifyitems(config, items):
-    excluded_tests = []
+    def get_excluded_tests(test_exclude_file):
```

Comment:
move this function out. It is better to use standalone functions instead nested.